tools/medenv/med_environ.xml

```diff
@@ -0,0 +1,141 @@
+<tool id="iabiodiv_smartbiodiv_med_environ" name="Adds the environment variables to a dataset" version="0.0.1" profile="22.05">
+    <description>
+        From the input dataset, it will request Copernicus for environment variables.
```

Comment:
the `name` + `description` will be shown in the toolbox together. The description is a bit cryptic.

maybe something like `name="Copernicus"` and `description="biodiversity environment variables"`

Reply:
Good point ,sorry for the quick and dirty name/description ; It should now be more explicit.